warning: in the working copy of 'C:\Users\hhyyl\Downloads\Stronghold-Protocol-v0.2.1\.scratch\audit-025\base-024\public\css\screens\game.css', LF will be replaced by CRLF the next time Git touches it
warning: in the working copy of 'C:\Users\hhyyl\Downloads\Stronghold-Protocol-v0.2.1\.scratch\audit-025\base-025\public\css\screens\game.css', LF will be replaced by CRLF the next time Git touches it
diff --git "a/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-024\\public\\css\\screens\\game.css" "b/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-025\\public\\css\\screens\\game.css"
index 190970d..515bb2e 100644
--- "a/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-024\\public\\css\\screens\\game.css"
+++ "b/C:\\Users\\hhyyl\\Downloads\\Stronghold-Protocol-v0.2.1\\.scratch\\audit-025\\base-025\\public\\css\\screens\\game.css"
@@ -683,0 +684,2 @@
+.ff-float--crit { color: #ff2a2a; }
+.ff-float--miss { color: #ff9000; }
